### PR TITLE
Added information about unit testing book in Books/Middle.md

### PR DESCRIPTION
**Pull request by albrt-dev**

### CODE REVIEW

File: Russian/Books/Middle.md [hunk 1]
```diff
@@ -48,6 +48,10 @@
 
     Краткое практическое пособие о том, как подходить к написанию программ посредством конечных автоматов. Наверно более простого и элегантного описания теории конечных автоматов и её практического применения будет сложно отыскать. Рекомендуем оизучить пару коммерческих работ, выполненных в описанной парадигме. Ссылки к исходному коду вы найдете в конце книги.
 
+- [Владимир Хориков - Принципы юнит-тестирования](https://www.ozon.ru/product/printsipy-yunit-testirovaniya-horikov-vladimir-211424826)
+
+   В книге подробно рассматриваются рекомендации, паттерны и антипаттерны, встречающиеся в области юнит-тестирования. После чтения этой книги вы будете знать все необходимое для того, чтобы стать экспертом в области создания успешных проектов — проектов, которые легко расширять и сопровождать благодаря хорошим тестам.
```
Comment (salmer) [inline Russian/Books/Middle.md:53]: Hi @albrt-dev. Please fix the repetitive part: "проектов — проектов,".

File: English/Books/Middle.md [hunk 2]
```diff
@@ -50,6 +50,10 @@
 
     This is a short and practical guide on how to approach programming using the theory of finite machines. You won't find a simpler and more elegant explanation of finite machine theory and its practical applications.
 
+- [Vladimir Khorikov - Unit Testing Principles, Practices, and Patterns: Effective testing styles, patterns, and reliable automation for unit testing, mocking, and integration testing with examples in C#](https://www.amazon.com/Unit-Testing-Principles-Practices-Patterns-ebook/dp/B09782L692)
+
+   The book details the recommendations, patterns, and anti-patterns found in the field of unit testing. After reading this book, you will know everything you need to be an expert at creating successful projects—projects that are easy to scale up and maintain with good tests
```
Comment (salmer) [inline English/Books/Middle.md:55]: The Google translate is not acceptable :)

I reworked it in the following way, please update it:

"The book provides a detailed overview of recommendations, patterns, and anti-patterns commonly encountered in the field of unit testing. After reading this book, you will know everything you need to become an expert in creating successful projects that are easily scalable and maintainable thanks to good tests."

**Review by salmer — CHANGES_REQUESTED** [review]

File: Russian/Books/Middle.md (hunk 1 above)
Comment (albrt-dev) [reply Russian/Books/Middle.md]: Updated.

**Comment by salmer** [comment]
@albrt-dev looks good, thank you for your contribution

**Review by salmer — APPROVED** [review]

**Pull request merged**